Admin UI: Add visual prop to Page header component (#76469)
WordPress/gutenberg: packages/admin-ui/CHANGELOG.md

## Unreleased

### New Features

> -   `Page`: Add `visual` prop to render an icon or image alongside the header title or breadcrumbs, outside the `h2` tag. [#76469](https://github.com/WordPress/gutenberg/pull/76469)

Reviewer: ```suggestion
-   `Page`: Add `visual` prop to render a decorative-only icon or image alongside the header title or breadcrumbs. [#76469](https://github.com/WordPress/gutenberg/pull/76469)
```
